Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -642,6 +643,7 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
     ///   - completion: Called when saving the message to the local DB finishes.
     ///
     public func createNewMessage(
+        messageId: MessageId? = nil,
```

Comment:
Nit: Maybe it makes more sense to put messageId after the text? 🤔 Since most of the time, nobody will be interested populating the messageId 🤔

Reply:
Id is always the first thing when creating a model, I think it makes sense